resources/views/components/layout.blade.php

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>{{__('store-layout.mismatch-finder')}}</title>
+    <link rel="stylesheet" href="{{ asset('css/app.css') }}">
+</head>
+<body>
+    <header>
+        <h1>{{__('store-layout.mismatch-finder')}}</h1>
+        <div class="username">
+            @auth
+                <a href='https://www.wikidata.org/wiki/User:{{ Auth::user()->username}}'><img src="/svg/user.svg" class="icon-user" />{{ Auth::user()->username }}</a><a href="{{ route('logout') }}">Logout</a>
+            @else
+                <a href="{{ route('login') }}">Log in</a>
+            @endauth
+        </div>
+    </header>
+    <nav class="tabs">
+        <ul role="tablist" aria-label="{{__('store-layout.aria-labels:tabs')}}">
+            <li role="tab" aria-selected="{{url()->current() == route('token') ? true : false}}"><a tabindex="-1" href="{{ route('token') }}" tabindex="0" class="{{url()->current() == route('token') ? 'selected' : ''}}"><a tabindex="-1" href="{{ route('token') }}">{{__('store-layout.tab:authentication-token')}}</a></li>
+            <li role="tab" aria-selected="false" tabindex="0"><a tabindex="-1" href="#">{{__('store-layout.tab:import-status')}}</a></li>
```

Comment:
To sum up: since we're finally taking a different, simpler approach, we should remove:

- `role="tablist"`
- `role="tab"`
- `aria-selected`attributes
- `tabindex`attributes

And add `aria-current="page"` to the currently selected link to let users know in which section they are.